cmd/commands/account/command.go

```diff
@@ -0,0 +1,261 @@
+func NewCommand() *cli.Command {
+	var cmd *command
+
+	return &cli.Command{
+		Name:        CommandName,
+		Usage:       "Manage your account",
+		Description: "Using account subcommands you can manage your account details and get information about it",
+		Before: func(ctx *cli.Context) error {
+			if err := clicontext.LoadUserConfigQuietly(ctx); err != nil {
+				return err
+			}
+			config.ParseFlagsNode(ctx)
+			nodeOptions := node.GetOptions()
+
+			tc := tequilapi_client.NewClient(nodeOptions.TequilapiAddress, nodeOptions.TequilapiPort)
+			cmd = &command{tequilapi: tc}
+
+			return nil
+		},
+		Subcommands: []*cli.Command{
+			{
+				Name:  "register",
+				Usage: "Submit a registration request",
+				Action: func(ctx *cli.Context) error {
+					cmd.register()
+					return nil
+				},
+			},
+			{
+				Name:  "topup",
+				Usage: "Create a new top up for your account",
+				Flags: []cli.Flag{&flagAmount, &flagCurrency},
+				Action: func(ctx *cli.Context) error {
+					cmd.topup(ctx)
+					return nil
+				},
+			},
+			{
+				Name:  "info",
+				Usage: "Display information about your account",
+				Flags: []cli.Flag{&flagLastTopup},
+				Action: func(ctx *cli.Context) error {
+					cmd.info(ctx)
+					return nil
+				},
+			},
+		},
+	}
+}
+
+type command struct {
+	tequilapi *tequilapi_client.Client
+}
+
+func (c *command) info(ctx *cli.Context) {
+	id, err := c.tequilapi.CurrentIdentity("", "")
+	if err != nil {
+		clio.Error("Failed to display information: could not get current identity")
+		return
+	}
+
+	clio.Status("SECTION", "General account information:")
+	c.infoGeneral(id.Address)
+
+	if ctx.Bool(flagLastTopup.Name) {
+		clio.Status("SECTION", "Last topup information:")
+		c.infoTopUp(id.Address)
+	}
+}
+
+func (c *command) topup(ctx *cli.Context) {
+	currencies, err := c.tequilapi.OrderCurrencies()
+	if err != nil {
+		clio.Error("Could not get a list of supported currencies")
+		return
+	}
+
+	currency := ctx.String(flagCurrency.Name)
+	if !matchString(currency, currencies) {
+		clio.Warn("Given currency cannot be used")
+		clio.Info("Supported currencies are:", strings.Join(currencies, ", "))
+		return
+	}
+
+	amount := ctx.Float64(flagAmount.Name)
+	if amount <= 0 {
+		clio.Warn("Amount to top up must be given and must be more than 0")
+		return
+	}
+
+	id, err := c.tequilapi.CurrentIdentity("", "")
+	if err != nil {
+		clio.Warn("Could not get your identity")
+		return
+	}
+
+	ok, err := c.identityIsUnregistered(id.Address)
+	if err != nil {
+		clio.Error(err.Error())
+		return
+	}
+
+	if ok {
+		clio.Warn("Please initiate your identity registration before trying top up")
+		return
+	}
+
+	resp, err := c.tequilapi.OrderCreate(identity.FromAddress(id.Address), contract.OrderRequest{
+		MystAmount:       amount,
+		PayCurrency:      currency,
+		LightningNetwork: false,
+	})
+
+	if err != nil {
+		clio.Error("Failed to create an top up request, make sure your requested amount is equal or more than 0.0001 BTC")
```

Comment:
Hm, why does it fixed to 0.0001 BTC? 
```suggestion
		clio.Error("Failed to create an top up request, make sure your requested amount is equal or more than 0.0001 BTC")
```

Reply:
That not my problem. Thats coingate doing coingate things. Minimum amount of myst is whatever it is converted to 0.0001 BTC at the time of purchase.